app/src/main/res/values/strings.xml

```diff
@@ -704,6 +705,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_recycling_type_plastic_packaging">Plastic packaging only</string>
     <string name="quest_recycling_type_scrap_metal">Scrap metal</string>
     <string name="quest_recycling_type_any_glass">Any glass</string>
-    <string name="quest_recycling_glass_title">Can only glass bottles be recycled here, or any type of glass?</string>
+    <string name="quest_recycling_glass_title">Can only glass bottles and jars be recycled here, or any type of glass?</string>
+    <string name="quest_determineRecyclingGlass_description_any_glass">Examples of glass that is often not accepted: tempered glass, window panes, glassware, light bulbs or mirrors.</string>
```

Comment:
"Examples" is plural, so change "that is often" to "that are often".

Reply:
Fixed. I thought that sentence construction would be

    (examples) of (glass that is often not accepted)

not

    (example of glass) (that are often not accepted)

Reply:
Your alternative sentence construction (with "glass" as a collective noun or singular noun) is, at first glance, valid. But the terms after the colon are majority plural, so a substitution (e.g. "light bulbs is not accepted") breaks it, so to speak. :)